presto-hive/src/test/java/io/prestosql/plugin/hive/rubix/TestRubixCaching.java

```diff
@@ -183,6 +183,10 @@ private void initializeRubix(RubixConfig rubixConfig, List<Node> nodes)
                 Optional.empty());
         rubixConfigInitializer = new RubixConfigurationInitializer(rubixInitializer);
         rubixInitializer.initializeRubix();
+        retry().run("wait for rubix to startup", () -> {
```

Comment:
how many retries will it do. What is the delay?

Reply:
see `io.prestosql.plugin.hive.util.RetryDriver#RetryDriver()` for defaults. By default there should be 10 attempts separated with 1s

Reply:
show work for this.